src/object/transform.h

```diff
@@ -163,6 +164,7 @@ typedef struct tf_ct_class
 #define CTV_PARTITION_NAME         "db_partition"
 #define CTV_DB_COLLATION_NAME      "db_collation"
 #define CTV_DB_CHARSET_NAME	   "db_charset"
+#define CTV_DB_SERVER_NAME         "db_server"
```

Comment:
How about 'db_link_server'?

Reply:
Since it is a system table to support the sql standard create server syntax, and it can be used as a Federated Table concept, doesn't it feel like the scope of use is narrowed to use the prefix with db_link ?

Reply:
For now, the name will remain as it is.